utils/src/main/kotlin/Utils.kt

```diff
@@ -391,6 +392,18 @@ fun JsonNode?.textValueOrEmpty(): String = this?.textValue()?.let { it } ?: ""
  */
 fun String.encodeOrUnknown() = fileSystemEncode().takeUnless { it.isBlank() } ?: "unknown"
 
+/*
```

Comment:
Please sort this in alphabetically to the other `JsonNode?` extension functions.

Reply:
Done.